core/domain/html_cleaner.py

```diff
@@ -114,8 +114,16 @@ def clean(user_submitted_html: str) -> str:
 
     # TODO(sll): Alert the caller if the input was changed due to this call.
     # TODO(sll): Add a log message if bad HTML is detected.
+
+    # Here we use MyPy ignore because core_tags comes from external registry
```

Comment:
Can we make the "external registry function" return Dict[str, List[str]] instead?